src/OSmOSE/utils/audio_utils.py

```diff
@@ -1,22 +1,42 @@
 from OSmOSE.config import SUPPORTED_AUDIO_FORMAT
 from pathlib import Path
-import glob
-import os
 
 
-def is_audio(filename: Path):
-    return any([filename.endswith(ext) for ext in SUPPORTED_AUDIO_FORMAT])
+def is_audio(filename: Path) -> bool:
+    """
+    Check if a given file is a supported audio file based on its extension.
 
+    Parameters
+    ----------
+    filename : Path
+        The path to the file to be checked.
 
-def get_audio_file(file_path: str | Path) -> list[Path]:
+    Returns
+    -------
+    bool
+        True if the file has an extension that matches a supported audio format,
+        False otherwise.
+    """
+    return filename.suffix in SUPPORTED_AUDIO_FORMAT
 
-    assert any(
-        [isinstance(file_path, str), isinstance(file_path, Path)]
-    ), "A Path or string must be provided"
 
+def get_audio_file(directory: Path) -> list[Path]:
```

Comment:
The function name is misleading, I'd rename it to something like `get_all_audio_files`